sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/implementation/Utility.java

```diff
@@ -49,4 +50,19 @@ private static ByteBuffer toByteArray(InputStream in) {
             throw LOGGER.logExceptionAsError(new RuntimeException(e));
         }
     }
+
+    /**
+     * Extracts the result ID from the URL.
+     *
+     * @param operationLocation The URL specified in the 'Operation-Location' response header containing the
+     * resultId used to track the progress and obtain the result of the analyze operation.
+     *
+     * @return The resultId used to track the progress.
+     */
+    public static String parseModelId(String operationLocation) {
+        if (!CoreUtils.isNullOrEmpty(operationLocation)) {
+            return operationLocation.substring(operationLocation.lastIndexOf('/') + 1);
```

Comment:
What happens if there isn't a `/` in the String?

Reply:
Updated to throw an error if `/` is not found in the string.